docs/src/reference/workflow-creation.md

```diff
@@ -0,0 +1,89 @@
+# Workflow Instance Creation
+
+Depending on the workflow definition, an instance of it can be created in various ways.
+
+## Explicit creation of a workflow instance
```

Comment:
```suggestion
## Create Workflow Instance Command
```

Using common terms from the API/clients could help users to understand it faster.